__tests__/app.test.js

```diff
@@ -44,13 +44,142 @@ describe('GET /api/topics', () => {
       });
   });
 });
-describe('GET /api/topisc', () => {
+describe('ERROR GET /api/topicd', () => {
```

Comment:
Line 47 doesn't match line 50... I'm being super picky here but it's good to keep the details straight!